keyboards/dz60/keymaps/billiams_layout4/build_flash.sh

```diff
@@ -0,0 +1,9 @@
+# dfu-programmer atmega32u4 erase --force
+# dfu-programmer atmega32u4 flash /path/to/firmware.hex
+# dfu-programmer atmega32u4 reset
+
+# run this in the qmk_firmware directory
+make dz60:billiams_layout4
+dfu-programmer atmega32u4 erase --force && \
+dfu-programmer atmega32u4 flash dz60_billiams_layout4.hex && \
+dfu-programmer atmega32u4 reset
```

Comment:
This can all be accomplished with `make dz60:billiams_layout4:flash`

Reply:
ok done thanks for the review